graph/GRAPH.h

```diff
@@ -0,0 +1,123 @@
+#include<iostream>
+#include<vector>
+#include<queue>
+#include<list>
+#include<tuple>
+#include<set>
+
+using namespace std;
```

Comment:
Explanations at tarjan.cpp, using namespace std; is even worse with headers, also this is C++ code, please don't use .h file extension
```suggestion
```